SVG/System76.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1024 1024">
+   <circle id="Background" cx="512" cy="512" r="512" style="fill:#595149"/>
+   <path transform="matrix(.5 0 0 .5 256 256)" d="M983.79 312.91A515.15 515.15 0 0 0 805.02 92.23l-.007.018.05.035.157.11-1.452 4.385c-9.797 18.218-120.24 217.92-161.82 293.1l-2.005 3.624a40919.505 40919.505 0 0 0-14.038 25.392c-32.137 55.496-46.885 117.96-39.456 167.09a160.84 160.84 0 0 0 27.982 69.229 166.8 166.8 0 0 0 108.69 67.899 172.66 172.66 0 0 0 13.689 1.763c5.063.453 10.232.681 15.368.681a169.65 169.65 0 0 0 102.47-33.979 161.58 161.58 0 0 0 63.1-102.78c7.305-43.414-3.291-87.187-29.845-123.26a171.08 171.08 0 0 0-110.09-67.363c-2.47-.411-5-.785-7.526-1.1a30.21 30.21 0 0 1-26.44-30.911 29.724 29.724 0 0 1 10.008-21.665 30.496 30.496 0 0 1 23.635-7.408c3.395.421 6.874.926 10.34 1.51 60.46 10.172 113.47 42.746 149.26 91.723 35.957 49.21 50.25 109.05 40.246 168.5-10.003 59.448-43.093 111.31-93.17 146.04a230.43 230.43 0 0 1-131.94 41.118c-6.852 0-13.751-.303-20.527-.9a239.894 239.894 0 0 1-18.594-2.394 226.57 226.57 0 0 1-148.33-92.923 238.49 238.49 0 0 1-18.194-30.163c-30.631-60.49-32.132-131.17-4.34-204.41 9.506-25.058 21.167-49.243 32.352-71.643l174.04-317.97 4.273-5.175.18.096.051-.103q-20.359-10.868-41.779-19.93a511.86 511.86 0 0 0-561.17 833.6q2.529 2.53 5.092 5.011l278.54-516.42H136.47l52.119 87.772a27.234 27.234 0 0 1 3.287 21.09 29.141 29.141 0 0 1-28.005 21.824 27.314 27.314 0 0 1-23.43-12.835l-77.795-130.94a27.226 27.226 0 0 1-4.115-14.749l.024-1.131a27.922 27.922 0 0 1 27.651-27.04h395.02c7.177 0 15.87 2.587 20.218 6.02l.412.308a31.546 31.546 0 0 1 9.439 40.534l-301.61 559.4a31.501 31.501 0 0 1-5.958 7.819 511.44 511.44 0 0 0 606.31 7.725h-239.84a31.76 31.76 0 0 1-31.76-31.76v-.642a31.76 31.76 0 0 1 31.76-31.76h313.38a512.48 512.48 0 0 0 100.2-551.4z" style="stroke-width:1.0286;fill:#fff" />
```

Comment:
- Remove `transform="matrix(.5 0 0 .5 256 256)"`
- Change the `style="stroke-width:1.0286;fill:#fff"` ➜ `style="fill:#fff"`
- Remove space before `/>`